### ISSUE TITLE
Mixture and MixtureSameFamily not available in tfp.distributions

### ISSUE THREAD

**Issue by dave-fernandes** [tensorflow/probability#384]
The following code (Colab) produces the error below:

```
import tensorflow.distributions as tfd

mix = 0.5
mixture = tfd.Mixture(
  cat=tfd.Categorical(prob=[mix, 1. - mix]),
  components=[tfd.Normal(loc=0., scale=1.), tfd.Normal(loc=0., scale=7.)])
```

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-1-646e8a5e2788> in <module>()
      2 
      3 mix = 0.5
----> 4 mixture = tfd.Mixture(
      5   cat=tfd.Categorical(prob=[mix, 1. - mix]),
      6   components=[tfd.Normal(loc=0., scale=1.), tfd.Normal(loc=0., scale=7.)])

AttributeError: module 'tensorflow.distributions' has no attribute 'Mixture'
SEARCH STACK OVERFLOW
```

Same error occurs when creating a MixtureSameFamily

**Comment by davmre** [comment]
You need that import line to be `from tensorflow_probability import
distributions as tfd`, or similar.

**Comment by csuter** [comment]
Sorry for the confusion Dave. We used to maintain the distributions library in TF and `tensorflow.contrib.distributions`. More recently we moved out into our own GH repo (this one) and pip package(s) (`tensorflow-probability` and `tfp-nightly`), but we haven't removed what was there in TF, so as not to break existing customers.

The Distributions you seek can be found by installing either the latest stable release of TFP -- `pip install tensorflow-probability`, version 0.6 which requires TF 1.13.1 -- , or the nightly build -- `pip install tfp-nightly`, which requires `tf-nightly`), which has more bleeding-edge features but weaker stability guarantees (ie, maybe bugs in newer code, new APIs subject to tweaking).
